src/main/java/org/sonar/plugins/stash/issue/collector/StashCollector.java

```diff
@@ -50,6 +52,55 @@ public static StashCommentReport extractComments(String jsonBody) throws StashRe
     return result;
   }
 
+  public static StashPullRequest extractPullRequest(String project, String repository, String pullRequestId, String jsonBody) throws StashReportExtractionException {
+    StashPullRequest result = new StashPullRequest(project, repository, pullRequestId);
+    
+    try {
+      JSONObject jsonPullRequest = (JSONObject) new JSONParser().parse(jsonBody);
+
+      long version = (long) jsonPullRequest.get("version");
+      result.setVersion(version);
+    
+      JSONArray jsonReviewers = (JSONArray) jsonPullRequest.get("reviewers");
+      if (jsonReviewers != null) {
+        for (Object objReviewer : jsonReviewers.toArray()) {
+          JSONObject jsonReviewer = (JSONObject) objReviewer;
+          
+          JSONObject jsonUser = (JSONObject) jsonReviewer.get("user");
+          if (jsonUser != null){
+            long id = (long) jsonUser.get("id");
+            String name = (String) jsonUser.get("name");
+            String slug = (String) jsonUser.get("slug");
+            String email = (String) jsonUser.get("email");

```

Comment:
regqrding REST API and if i am not wrong, the key for emqil is emailAddress and not email (https://developer.atlassian.com/static/rest/stash/3.11.3/stash-rest.html#idp157520).
IN this case, this call can be factorized with extractUser() method.